package.json

```diff
@@ -13,22 +13,26 @@
     "./compat": {
       "require": "./compat/dist/compat.js",
       "import": "./compat/dist/compat.module.js",
-      "browser": "./compat/dist/compat.umd.js"
+      "browser": "./compat/dist/compat.module.js",
+      "umd": "./compat/dist/compat.umd.js"
     },
     "./debug": {
-      "require": "./debug/dist/debug.js",
+      "require": "./compat/dist/debug.js",
```

Comment:
Is that change from `./debug` to `./compat` correct ?

Reply:
Thanks!